### PR TITLE
CLDR-15473 drop section numbering

### PR DESCRIPTION
**Pull request by srl295**
CLDR-15473

Note: more radical changes to personNames as it is newer, simplified its header structure.

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

File: docs/ldml/tr35-collation.md [hunk 1]
```diff
@@ -310,17 +310,17 @@ Comments with DUCET-style weights in files other than allkeys_CLDR.txt and allke
...
-### 2.6 <a name="Root_Data_File_Formats" href="#Root_Data_File_Formats">Root Collation Data File Formats</a>
+### <a name="Root_Data_File_Formats" href="#Root_Data_File_Formats">Root Collation Data File Formats</a>
 
 The file formats may change between versions of CLDR. The formats for CLDR 23 and beyond are as follows. As usual, text after a # is a comment.
 
-#### 2.6.1 <a name="File_Format_allkeys_CLDR_txt" href="#File_Format_allkeys_CLDR_txt">allkeys_CLDR.txt</a>
+#### <a name="File_Format_allkeys_CLDR_txt" href="#File_Format_allkeys_CLDR_txt">allkeys_CLDR.txt</a>
 
-This file defines CLDR’s tailoring of the DUCET, as described in _Section 2, [Root Collation](#Root_Collation)_ .
+This file defines CLDR’s tailoring of the DUCET, as described in _, [Root Collation](#Root_Collation)_ .
```
Comment (macchiati) [inline docs/ldml/tr35-collation.md:319]: Spacing and stray comma problems

File: docs/ldml/tr35-collation.md [hunk 2]
```diff
@@ -520,11 +520,11 @@ The final table gives certain hard-coded byte values. The "trail" area is provid
 
 > 👉 **Note**: The particular primary lead bytes for Hani vs. IMPLICIT vs. TRAILING are only an example. An implementation is free to move them if it also moves the explicit TRAILING weights. This affects only a small number of explicit mappings in FractionalUCA.txt, such as for U+FFFD, U+FFFF, and the “unassigned first primary”. It is possible to use no SPECIAL bytes at all, and to use only the one primary lead byte FF for TRAILING weights.
 
-#### 2.6.3 <a name="File_Format_UCA_Rules_txt" href="#File_Format_UCA_Rules_txt">UCA_Rules.txt</a>
+#### <a name="File_Format_UCA_Rules_txt" href="#File_Format_UCA_Rules_txt">UCA_Rules.txt</a>
 
-The format for this file uses the CLDR collation syntax, see _Section 3, [Collation Tailorings](#Collation_Tailorings)_.
+The format for this file uses the CLDR collation syntax, see _, [Collation Tailorings](#Collation_Tailorings)_.
```
Comment (macchiati) [inline docs/ldml/tr35-collation.md:525]: Same here. Seemed like general issue

File: docs/ldml/tr35-collation.md [hunk 3]
```diff
@@ -987,7 +987,7 @@ _Examples:_
 | `en-u-kr-arab-cyrl-others-symbol` | Reorder Arabic characters first, then Cyrillic, and put symbols at the end—after all other characters. |
 | `en-u-kr-others`                  | Remove any locale-specific reordering, and use DUCET order for reordering blocks. |
 
-The default reordering groups are defined by the FractionalUCA.txt file, based on the primary weights of associated collation elements. The file contains special mappings for the start of each group, script, and reorder-reserved range, see _Section 2.6.2, [FractionalUCA.txt](#File_Format_FractionalUCA_txt)_.
+The default reordering groups are defined by the FractionalUCA.txt file, based on the primary weights of associated collation elements. The file contains special mappings for the start of each group, script, and reorder-reserved range, see _, [FractionalUCA.txt](#File_Format_FractionalUCA_txt)_.
```
Comment (macchiati) [inline docs/ldml/tr35-collation.md:990]: Ditto. I'm not going to try to find all these manually

**Review by macchiati — CHANGES_REQUESTED** [review]

File: docs/ldml/tr35-collation.md (hunk 1 above)
Comment (srl295) [reply docs/ldml/tr35-collation.md:319]: good catch.  The italicized links were a little more challenging, i will go back and check them.

File: docs/ldml/tr35-collation.md [hunk 4]
```diff
@@ -222,7 +222,7 @@ The following table shows the differences. When emoji ordering is supported, the
 
 ## <a name="Root_Collation" href="#Root_Collation">Root Collation</a>
 
-The CLDR root collation order is based on the [Default Unicode Collation Element Table (DUCET)](https://www.unicode.org/reports/tr10/#Default_Unicode_Collation_Element_Table) defined in _UTS #10: Unicode Collation Algorithm_ [[UCA](https://www.unicode.org/reports/tr41/#UTS10)]. It is used by all other locales by default, or as the base for their tailorings. (For a chart view of the UCA, see Collation Chart [[UCAChart](tr35.md#UCAChart)].)
+The CLDR root collation order is based on the [Default Unicode Collation Element Table (DUCET)](https://www.unicode.org/reports/tr10/#Default_Unicode_Collation_Element_Table) defined in _UTS #10: Unicode Collation Algorithm_[[UCA](https://www.unicode.org/reports/tr41/#UTS10)]. It is used by all other locales by default, or as the base for their tailorings. (For a chart view of the UCA, see Collation Chart [[UCAChart](tr35.md#UCAChart)].)
```
Comment (macchiati) [inline docs/ldml/tr35-collation.md:225]: The space probably needs to remain, so we don't have ... _Algorithm_[UCA]

File: docs/ldml/tr35-dates.md [hunk 5]
```diff
@@ -1326,7 +1326,7 @@ For examples, see [Day Periods Chart](https://unicode-org.github.io/cldr-staging
 <!ELEMENT exemplarCity ( #PCDATA ) >
 ```
 
-The time zone IDs (TZID) are language-independent, and follow the _TZ time zone database_ [[Olson](tr35.md#Olson)] and naming conventions. However, the display names for those IDs can vary by locale. The generic time is so-called _wall-time_; what clocks use when they are correctly switched from standard to daylight time at the mandated time of the year.
+The time zone IDs (TZID) are language-independent, and follow the _TZ time zone database_[[Olson](tr35.md#Olson)] and naming conventions. However, the display names for those IDs can vary by locale. The generic time is so-called _wall-time_; what clocks use when they are correctly switched from standard to daylight time at the mandated time of the year.
```
Comment (macchiati) [inline docs/ldml/tr35-dates.md:1329]: Here as well.

File: docs/ldml/tr35-dates.md [hunk 6]
```diff
@@ -1395,7 +1395,7 @@ They may designate countries or territories; their actual capital city may be a
 
 > **Note:** Transmitting "14:30" with no other context is incomplete unless it contains information about the time zone. Ideally one would transmit neutral-format date/time information, commonly in UTC (GMT), and localize as close to the user as possible. (For more about UTC, see [[UTCInfo](tr35.md#UTCInfo)].)
 
-The conversion from local time into UTC depends on the particular time zone rules, which will vary by location. The standard data used for converting local time (sometimes called _wall time_) to UTC and back is the _TZ Data_ [[Olson](tr35.md#Olson)], used by Linux, UNIX, Java, ICU, and others. The data includes rules for matching the laws for time changes in different countries. For example, for the US it is:
+The conversion from local time into UTC depends on the particular time zone rules, which will vary by location. The standard data used for converting local time (sometimes called _wall time_) to UTC and back is the _TZ Data_[[Olson](tr35.md#Olson)], used by Linux, UNIX, Java, ICU, and others. The data includes rules for matching the laws for time changes in different countries. For example, for the US it is:
```
Comment (macchiati) [inline docs/ldml/tr35-dates.md:1398]: And here

File: docs/ldml/tr35-general.md [hunk 7]
```diff
@@ -451,7 +451,7 @@ The `lineOrder` and `characterOrder` elements specify the default general orderi
...
 
-If the value of lineOrder is one of the vertical values, then the value of characterOrder must be one of the horizontal values, and vice versa. For example, for English the lines are top-to-bottom, and the characters are left-to-right. For Mongolian (in the Mongolian Script) the lines are right-to-left, and the characters are top to bottom. This does not override the ordering behavior of bidirectional text; it does, however, supply the paragraph direction for that text (for more information, see _UAX #9: The Bidirectional Algorithm_ [[UAX9](https://www.unicode.org/reports/tr41/#UAX9)]).
+If the value of lineOrder is one of the vertical values, then the value of characterOrder must be one of the horizontal values, and vice versa. For example, for English the lines are top-to-bottom, and the characters are left-to-right. For Mongolian (in the Mongolian Script) the lines are right-to-left, and the characters are top to bottom. This does not override the ordering behavior of bidirectional text; it does, however, supply the paragraph direction for that text (for more information, see _UAX #9: The Bidirectional Algorithm_[[UAX9](https://www.unicode.org/reports/tr41/#UAX9)]).
```
Comment (macchiati) [inline docs/ldml/tr35-general.md:454]: And here

**Review by macchiati — CHANGES_REQUESTED** [review]
Almost there.

**Comment by srl295** [comment]
ptal

Lots of inconsistent citation styles.

```md
- [_[Modifiers](#modifiers)_] 
- **_[Inheritance vs Related Information](tr35.md#Inheritance_vs_Related)_**.
- [_Supplemental_ Metadata](#Appendix_Supplemental_Metadata)
- See [[Fields](#fields)]
```

**Review by macchiati — APPROVED** [review]

**Pull request merged**